code/modules/antagonists/heretic/magic/aggressive_spread.dm

```diff
@@ -17,7 +17,16 @@
 	aoe_radius = 3
 
 /datum/action/cooldown/spell/aoe/rust_conversion/get_things_to_cast_on(atom/center)
-	return RANGE_TURFS(aoe_radius, center)
+
+	var/list/things_to_convert = RANGE_TURFS(aoe_radius, center)
+
+	// Also converts things right next to you.
+	for(var/atom/movable/nearby_movable in view(1, center))
+		if(nearby_movable == owner)
+			continue
+		things_to_convert += nearby_movable
```

Comment:
This seems questionable. Like being able to destroy every silicon and mech and beepsky adjacent to you with no limit?

Reply:
featuren't

how do i make sure rusted harvesters can rust things other than turfs then. i want them to convert altarchivorges

Reply:
Maybe just add `if(!isstructure(nearby_movable)) continue`